test/WebJobs.Script.Tests/UtilityTests.cs

```diff
@@ -86,7 +86,7 @@ public void TryMatchAssembly_ReturnsExpectedResult()
             sw.Start();
             await Utility.DelayWithBackoffAsync(2, CancellationToken.None);
             sw.Stop();
-            Assert.True(sw.ElapsedMilliseconds >= 2000);
+            Assert.True(sw.ElapsedMilliseconds >= 2000, $"Expected >= 2000; Actual: {sw.ElapsedMilliseconds}");
```

Comment:
I've seen this be pretty flaky so decided to add some additional logging to see what the problem may be.